From f439198dbc7096b495332fd62b87f38f4e96afde Mon Sep 17 00:00:00 2001
From: Simone Schwizer <simone.schwizer@bytesatwork.ch>
Date: Thu, 14 Nov 2024 14:08:15 +0100
Subject: [PATCH] bytesatwork: bawconfig: Remove common.h includes

This resolves the checkpatch ERROR: Do not add common.h to files.

Signed-off-by: Simone Schwizer <simone.schwizer@bytesatwork.ch>
(cherry picked from commit ce864843561f63a34b00fa1237ffe0959c31485f)
---
 board/bytesatwork/baw_config/baw_config_eeprom.c | 1 -
 board/bytesatwork/baw_config/cmd_baw_config.c    | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/board/bytesatwork/baw_config/baw_config_eeprom.c b/board/bytesatwork/baw_config/baw_config_eeprom.c
index da577e2d2e..d5ae831b2f 100644
--- a/board/bytesatwork/baw_config/baw_config_eeprom.c
+++ b/board/bytesatwork/baw_config/baw_config_eeprom.c
@@ -6,7 +6,6 @@
 #include "baw_config_eeprom.h"
 #include "baw_config.h"
 
-#include <common.h>
 #include <dm/uclass.h>
 #include <i2c.h>
 #include <hexdump.h>
diff --git a/board/bytesatwork/baw_config/cmd_baw_config.c b/board/bytesatwork/baw_config/cmd_baw_config.c
index 922f021701..b1076d424c 100644
--- a/board/bytesatwork/baw_config/cmd_baw_config.c
+++ b/board/bytesatwork/baw_config/cmd_baw_config.c
@@ -3,9 +3,9 @@
  * Copyright (C) 2015 bytesatwork AG - https://www.bytesatwork.io
  */
 
-#include <common.h>
 #include <command.h>
 #include <post.h>
+#include <vsprintf.h>
 #include "baw_config_builtin.h"
 #include "baw_config_eeprom.h"
 
